src/header/header.scss

```diff
@@ -265,4 +292,124 @@
       }
     }
   }
+
+  .iui-split-header-button {
```

Comment:
Why is this class needed? It's not even a modifier on base split-button so you have to reinvent the wheel

Reply:
It is not really worth reusing split button because you need to override a lot of stuff. Previously I thought the same but then I tried to do that myself 😃

Reply:
The main reason it became complicated is that we wanted the split button to not act split, but more like a secondary button between a primary button.

It is much much simpler if we just change the intended hover effect to this:
| hover left | hover right |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/9084735/128199701-74e02beb-f3fd-47a3-96fc-8012b6f49f8a.png) | ![image](https://user-images.githubusercontent.com/9084735/128199799-7411ae41-94ed-4dea-9759-092d77de9b8c.png) |

@FlyersPh9 Is there anything wrong with this hover effect? I think it looks better and is significantly easier to implement (2 lines of css vs 100+ lines).